packages/app/src/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export { default as RainbowFirebaseApp } from './components/App';
+export { default as SideBarNavigation } from './components/SideBarNavigation';
+export { default as SideBarOption } from './components/SideBarOption';
```

Comment:
here we need try to find a way we can add the interfaces of:
    showAppSpinner,
    hideAppSpinner,
    showAppMessage,
    confirmModal,
    showAppNotification,
    
 we use this methods, and it useful to know what props it will receive when using them